chrome-launcher/README.md

```diff
@@ -125,5 +126,7 @@ before_script:
   - export DISPLAY=:99.0
   - export CHROME_PATH="$(pwd)/chrome-linux/chrome"
   - sh -e /etc/init.d/xvfb start
-  - curl -L https://raw.githubusercontent.com/GoogleChrome/lighthouse/v2.1.0/lighthouse-core/scripts/download-chrome.sh | bash
+  - sleep 3 # wait for xvfb to boot
```

Comment:
I believe @paulirish recently removed this line as unnecessary, not sure if we should encourage using it more?

thoughts paul?

Reply:
i removed it because we dont need to use chrome in the next line of travis `script`... we have at least 3 seconds worth of work before we get there.

but i can imagine for other projects using chrome-launcher that they'd want this bit to be more reliable.